README.adoc

```diff
@@ -208,13 +209,13 @@ include::{common-includes}/os-tabs.adoc[]
 
 // static guide instructions:
 ifndef::cloud-hosted[]
-The services take some time to become available. 
-You can access the application by making requests to the `query/systemLoad` endpoint by going to http://localhost:9080/query/systemLoad[localhost:9080/query/systemLoad^].
+The services take some time to become available.  
+You can access the application by making requests to the `query/systemLoad` endpoint by going to http://localhost:9080/query/systemLoad[http://localhost:9080/query/systemLoad^].
```

Comment:
You can access the application by making requests to the `query/systemLoad` endpoint by going to the http://localhost:9080/query/systemLoad[http://localhost:9080/query/systemLoad^] URL.